acropolis-shell/src/main/kotlin/org/ephyra/acropolis/shell/LinkCommand.kt

```diff
@@ -0,0 +1,79 @@
+package org.ephyra.acropolis.shell
+
+import org.ephyra.acropolis.service.api.INetworkService
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.shell.standard.ShellComponent
+import org.springframework.shell.standard.ShellMethod
+
+@ShellComponent
+class LinkCommand {
+    private val Logger = LoggerFactory.getLogger(LinkCommand::class.java)
+
+    @Autowired
+    private lateinit var appState: AppState
+
+    @Autowired
+    private lateinit var networkService: INetworkService
+
+    @ShellMethod("Link two items")
```

Comment:
I know the end game isn't the shell but, this could be a bit more descriptive seeing that in all cases thus far this is linking *something* to a **network**, rather than just **two _things_**. 
Just makes our lives easier if we end up basing our REST services, or some other interface off of the code here, as this is the most fully fledged interaction we have. 

Especially seeing as any other `~linking~` behaviour we have should probably be a `connection`.

Reply:
Agree it's not a helpful description of the command. Maybe can improve it if more functionality is added to the method